xbmc/Application.cpp

```diff
@@ -1329,6 +1329,17 @@ void CApplication::StopPVRManager()
   g_EpgContainer.Stop();
 }
 
+bool CApplication::SearchMissingChannelIcons()
```

Comment:
please drop this method and do the necessary stuff in `CBuiltins` see my comment below